.gitignore

```diff
@@ -17,3 +17,12 @@ out.txt
 err.txt
 /korim/demo.png
 *.hprof
+/korvi/build/
+/luak/build/
+/kbox2d/build/
+/korge-box2d/build/
+/korge-dragonbones/build/
+/korge-fleks/build/
+/korge-spine/build/
+/korge-swf/build/
+/korge-testing/build/
```

Comment:
I guess you should remove those folders yourself, because they are not longer in the repo since they were extracted to kproject

Reply:
Done